contract/r/gnoswap/gov/governance/proposal_test.gno

```diff
@@ -284,11 +283,11 @@ func TestUpdateProposalsState(t *testing.T) {
 			proposals = avl.NewTree()
 
 			proposal := tt.setupProposal(tt.currentTime)
-			proposals.Set(fomatUint(1), proposal)
+			proposals.Set(formatUint(uint32(1)), proposal)
 
 			updateProposalsState()
 
-			updatedProposal, exists := proposals.Get(fomatUint(1))
+			updatedProposal, exists := proposals.Get(formatUint(uint32(1)))
```

Comment:
also here
```suggestion
			updatedProposal, exists := proposals.Get(formatUint(1))
```

Reply:
It needs to be, if you run test file you can see it panics without converting